templates/api/api_key_confirm.html

```diff
@@ -7,6 +7,7 @@
                 <div class="card-body">
                     <h5 class="card-title">Email Login Request</h5>
                     <form method="POST">
+                        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
```

Comment:
this not needed as:
 - api_endpoint.py is exempt from csrf anyhow
 - form is protected by unique code in url

Reply:
removed the csrf_token line.